src/test/java/seedu/address/model/widget/WidgetModelManagerTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.widget;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+import static seedu.address.testutil.TypicalClients.AMY;
+import static seedu.address.testutil.TypicalClients.BOB;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.client.Client;
+import seedu.address.testutil.ClientBuilder;
+
+
+
+class WidgetModelManagerTest {
```

Comment:
```suggestion
public class WidgetModelManagerTest {
```

Also, extra newlines above this can be removed